src/pages/workspace/tags/WorkspaceTagsPage.tsx

```diff
@@ -77,14 +79,13 @@ function WorkspaceTagsPage({policyTags, route}: WorkspaceTagsPageProps) {
         fetchTags();
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
-
     const policyTagLists = useMemo(() => PolicyUtils.getTagLists(policyTags), [policyTags]);
     const tagList = useMemo<PolicyForList[]>(
         () =>
             policyTagLists
                 .map((policyTagList) =>
-                    Object.values(policyTagList.tags || [])
-                        .sort((a, b) => localeCompare(a.name, b.name))
+                    lodashSortBy(Object.values(policyTagList.tags || []), 'name', localeCompare)
+                        .filter((value): value is OnyxCommon.OnyxValueWithOfflineFeedback<OnyxTypes.PolicyTag> => typeof value === 'object' && value !== null)
```

Comment:
I don't see this pattern throughout the app. No other way?

Reply:
I've resolved new conflict and replaced the pattern as well:

```js
                .map((policyTagList) =>
                    lodashSortBy(Object.values(policyTagList.tags || []), 'name', localeCompare).map((value) => {
                        const tag = value as OnyxCommon.OnyxValueWithOfflineFeedback<OnyxTypes.PolicyTag>;
```